### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by driverpt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+## 3.1.3
+  - Improved performance when cloning events with big arrays
```
Comment (jsvd) [inline CHANGELOG.md]: I suggest "Improved performance by no longer cloning the source field if it is the same as the target field"

Comment (driverpt) [inline CHANGELOG.md]: Addressed

**Comment by suyograo** [comment]
LGTM

**Comment by driverpt** [comment]
@suyograo , when is this going to be released ?

**Comment by jsvd** [comment]
A new version has been published, thanks for the contribution :+1:

**Pull request merged**